alembic/versions/5cb3ce4c3e39_rename_q30_metrics_column.py

```diff
@@ -0,0 +1,32 @@
+"""Rename q30 metrics column
+
+Revision ID: 5cb3ce4c3e39
+Revises: 201b16c45366
+Create Date: 2023-08-18 13:56:53.793003
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "5cb3ce4c3e39"
+down_revision = "201b16c45366"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "sample_lane_sequencing_metrics",
+        "sample_base_fraction_passing_q30",
+        new_column_name="sample_base_percentage_passing_q30",
```

Comment:
```suggestion
        new_column_name="base_percentage_passing_q30",
```
I think this could be informative enough, the rows are samples in the table.

Reply:
I thought that too, maybe better in a separate PR where we rename the other fields as well?

Reply:
yes sounds great! It is nothing super urgent anyway.